src/Theme/AboutPage.elm

```diff
@@ -428,9 +428,9 @@ makerHeaderStyle =
 makerLogoStyle : Style
 makerLogoStyle =
     batch
-        [ width (px 200)
+        [ height (px 50)
         , margin4 (rem 2) auto (rem 3) auto
-        , withMediaSmallDesktopUp [ width (px 250) ]
+        , withMediaSmallDesktopUp [ height (px 55) ]
```

Comment:
The new GI logo is a bit taller than the previous one, so constraining the height ensures the GFSC and GI logos align nicely when viewed side-by-side:

![Screenshot 2024-08-26 at 16 56 45](https://github.com/user-attachments/assets/8e15ada4-a52c-4bba-bab4-82255add90e6)